lms/static/certificates/vendor/bi-app/_variables-ltr.scss

```diff
@@ -0,0 +1,15 @@
+// ------------------------------------------
+// left to right variables to be used by bi-app mixins
+// authors: 
+// twitter.com/anasnakawa
+// twitter.com/victorzamfir
+// licensed under the MIT license 
+// http://www.opensource.org/licenses/mit-license.php
+// ------------------------------------------
+
+// namespacing variables with bi-app to
+// avoid conflicting with other global variables
+$bi-app-left 				    : left;
+$bi-app-right 				  : right;
+$bi-app-direction 			: ltr;
+$bi-app-invert-direction: rtl;
```

Comment:
It looks like our platform version of this leaves the file as is (no added extra line) - https://github.com/edx/edx-platform/blob/master/lms%2Fstatic%2Fsass%2Fvendor%2Fbi-app%2F_variables-ltr.scss. Cool to keep consistent with that?